public/data/hailer24.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Anmol Bhutani",
+  "type": "personal",
+  "bio": "Just trying to git gud",
+  "avatar": "https://github.com/hailer24.png",
+  "links": [
+    {
+      "name": "Connect with me on GitHub",
+      "url": "https://github.com/hailer24",
+      "icon": "github"
+    },
+    {
+      "name": "Conenct with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/anmol-bhutani-b132931a5/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Take a look at my portfolio",
+      "url": "https://mole24.netlify.app/",
+      "icon": "Link"
+    },
+    {
+      "name": "check my Codeforces account",
+      "url": "https://codeforces.com/profile/mole24",
+      "icon": "Codeforces"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Web development",
+      "date": "June 2021",
+      "icon": "Globe",
+      "color": "blue"
+    },
+    {
+      "title": "Joined NIT Kurukshetra",
+      "date": "July 2019",
+      "icon": "Graduation Hat",
```

Comment:
```suggestion
      "icon": "graduation-hat",
```

Reply:
This is an unnecessary change, both will work in the code as we convert everything to lowercase

Reply:
but the dash @emmalearnscode ?

Reply:
Ok, sorry, I missed that @krshdev .